CHANGELOG.md

```diff
@@ -10,7 +10,9 @@
 
 #### Enhancements
 
-* None.
+* Adds `noAsyncWithoutAwait` opt-in rule that check if a async function call at least one async function.  
```

Comment:
Do we want to fix warnings in this PR? There is so many of them it would be nice to do that in single PR.